m4/lx_find_gotcha.m4

```diff
@@ -0,0 +1,18 @@
+AC_DEFUN([X_AC_GOTCHA], [
```

Comment:
I'd just name this file gotcha.m4, and call the macro UNIFYCR_AC_GOTCHA as mentioned above.